src/Hazelcast.Net.Examples/Client/HostedExample.cs

```diff
@@ -107,8 +107,12 @@ public async Task StartAsync(CancellationToken cancellationToken)
             {
                 _logger.LogInformation("Starting...");
 
+                // Assume, Hazelcast is running on local.
+                var options = new HazelcastOptionsBuilder()
```

Comment:
Is this needed? The client should fall back to `127.0.0.1` automatically when no address is provided.

Reply:
It was using `StartNewClientAsync(CancellationToken)` which is deprecated. However, correction wasn't nice, and corrected&tested at https://github.com/hazelcast/hazelcast-csharp-client/pull/763/commits/d40907e6cdf75465a9527659bbb5ad40d8ea1987